### ISSUE TITLE
I use arch btw

### ISSUE THREAD

**Issue by KotMilkMeoWtwo** [overmighty/i-use-arch-btw#12]
![image](https://user-images.githubusercontent.com/88706776/176256061-ec2d6f3a-7f9e-44ec-899f-016b3a4e7177.png)

**Comment by overmighty** [comment]
Good.

**Comment by askorbinovaya-kislota** [comment]
btw, kde is only 737MB
![Screenshot_20220704_214056](https://user-images.githubusercontent.com/105155938/177205130-6b813192-fa09-4fa2-9295-aa6f76a01fc7.png)

**Comment by KotMilkMeoWtwo** [comment]
You launch only konsole. I launch Spotify, steam, discord, minecraft server. We are different. Xd

**Comment by terremoth** [comment]
> btw, kde is only 737MB 

737mb only for a doing-nothing desktop environment with only konsole opened.
This is a CRIME. 

K.I.S.S. FTW

I hope one day Budgie, Trinity and Enlightenment Desktop Environment become more popular, they are the only ones smaller that use just a few mb of ram. I can't accept it is 2022 and there are desktop environments using almost a fucking whole GB of ram.

**Comment by Vermillion-de** [comment]
lol!
Are there some projects allow us to make kde-plasma (or other desktop environments) like macOS automatically ? >_

**Comment by zhengliw** [comment]
Yeah, you know. I don't think it's a very bad thing. Hardware standards are evolving and I think it's okay for software to use a little more resources than in 2010....

But I **do** love simplicity and saving resources... Who doesn't? :)

Edit: Oh, also, @askorbinovaya-kislota nice desktop, by the way!

**Comment by askorbinovaya-kislota** [comment]
> Edit: Oh, also, @askorbinovaya-kislota nice desktop, by the way!

it is just very vanilla kde plasma with a nice wallpaper and color scheme stolen from old manjaro :)

i also agree that plasma does not require 260mb of ram for nothing (`plasmashell`'s memory usage), but it is still much better than other desktops

**Comment by 0xsweat** [comment]
I use i3wm btw

**Comment by Walker-00** [comment]
who use de in arch.
![xmonad](https://github.com/overmighty/i-use-arch-btw/assets/85013114/2665afa3-bf19-4c72-bb9a-a0e54d518ef8)

**Comment by Walker-00** [comment]
Who use DE in Arch?
![xmonad](https://github.com/overmighty/i-use-arch-btw/assets/85013114/6ce9b0c7-5002-476a-a427-5c2a63efc5b8)

**Comment by ghost** [comment]
> Who use DE in Arch?
> ![xmonad](https://github.com/overmighty/i-use-arch-btw/assets/85013114/6ce9b0c7-5002-476a-a427-5c2a63efc5b8)
> 

Cool customization! I use pure dwm btw.

**Comment by MannuVilasara** [comment]
> hyprland btw
> ![hyprland](https://github.com/overmighty/i-use-arch-btw/assets/117009138/ae6ab353-3204-4a91-b334-0c00bac65304)

**Comment by Walker-00** [comment]
bspwm

![mona_lisa](https://github.com/overmighty/i-use-arch-btw/assets/85013114/2b906bb3-fd62-4ea9-8319-bbc1897233a1)

**Comment by UnFamousSoul** [comment]
![image](https://github.com/overmighty/i-use-arch-btw/assets/83215457/05c5fd7c-b33c-4534-8b6c-10c99595087c)

**Comment by zhengliw** [comment]
I don't use Arch :(

**Comment by FlekGeKei** [comment]
![screen-1706465024](https://github.com/overmighty/i-use-arch-btw/assets/104369860/d1ad322b-2cd4-4381-922c-e5905fb8a71d)

**Comment by axurright** [comment]
I use Arch btw

**Comment by TuberAsk** [comment]
I use arch btw double combo
